lib/variables/variable-number.tpp

```diff
@@ -94,6 +95,22 @@ template<typename T> NumberVariable<T>::operator T() const
 	return GetValue();
 }
 
+template<typename T> NumberVariable<T> &NumberVariable<T>::operator--()
```

Comment:
To be honest I am not sure if those operators really make sense.
They change the selection to be a fixed value if it was previously a variable.
That side effect is not really clear from the operator.

Reply:
Left those by accident after testing something. I'll remove unless you have better approach, feel free to edit

Reply:
Please remove them in that case - thank you! :)